scrapy/commands/startproject.py

```diff
@@ -76,3 +74,9 @@ def run(self, args, opts):
         print("You can start your first spider with:")
         print("    cd %s" % project_name)
         print("    scrapy genspider example example.com")
+
+    @property
+    def templates_dir(self):
+        _templates_base_dir = self.settings['TEMPLATES_DIR'] or \
+            join(scrapy.__path__[0], 'templates')
+        return join(_templates_base_dir, 'project')
```

Comment:
I did this so as not to change the present convention (startproject looks for 'project' and genspider looks for 'spiders' inside template directory). Should I let the user define the path to the folder itself?